src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/EnvironmentHelper.cs

```diff
@@ -158,37 +160,47 @@ static ApplicationConfig ReadApplicationConfig (string envFile)
 						ret.have_runtime_config_blob = ConvertFieldToBool ("have_runtime_config_blob", envFile, i, field [1]);
 						break;
 
-					case 8: // bound_stream_io_exception_type: byte / .byte
+					case 8: // have_assemblies_blob: bool / .byte
```

Comment:
Given that every time we update `libxamarin-app.so` this `switch` block is updated, and some (many?) of the `case` labels changed… is there a more "dummy proof" data structure that can be used here?  A `List<Action>` or something?

Reply:
Why complicate/obfuscate matters if they can remain simple? The `switch` is self-documenting and clear in its purpose. Adding a level of indirection wouldn't give us any advantage IMO except for a smaller occasional diff.